src/main/resources/assets/createframed/lang/ru-ru.json

```
@@ -0,0 +1,217 @@
+{
+  "itemGroup.createframed.create_framed": "Create: Framed",
+
+  "block.createframed.copper_window": "Медное окно",
+  "block.createframed.copper_window_pane": "Медная оконная панель",
+  "block.createframed.zinc_window": "Цинковое окно",
+  "block.createframed.zinc_window_pane": "Цинковая оконная панель",
+  "block.createframed.brass_window": "Латунное окно",
+  "block.createframed.brass_window_pane": "Латунная оконная панель",
+  "block.createframed.rose_quartz_window": "Окно из розового кварца",
+  "block.createframed.rose_quartz_window_pane": "Оконная панель из розового кварца",
+  "block.createframed.andesite_alloy_window": "Окно из андезитного сплава",
+  "block.createframed.andesite_alloy_window_pane": "Оконная панель из андезитного сплава",
+  "block.createframed.cardboard_window": "Кортонное окно",
+  "block.createframed.cardboard_window_pane": "Картонная панель",
+
+  "block.createframed.red_stained_framed_glass": "Красное витражное обрамлённое стекло",
+  "block.createframed.orange_stained_framed_glass": "Оранжевое витражное обрамлённое стекло",
+  "block.createframed.yellow_stained_framed_glass": "Жёлтое витражное обрамлённое стекло",
+  "block.createframed.green_stained_framed_glass": "Зелёное витражное обрамлённое стекло",
+  "block.createframed.lime_stained_framed_glass": "Лаймовое витражное обрамлённое стекло",
+  "block.createframed.blue_stained_framed_glass": "Синее витражное обрамлённое стекло",
+  "block.createframed.light_blue_stained_framed_glass": "Голубое витражное обрамлённое стекло",
+  "block.createframed.cyan_stained_framed_glass": "Циановое витражное обрамлённое стекло",
```

All "Cyan" stuff should be "Бирюзовый(ая/ое)", not "Циановый(ая/ое)".
```suggestion
  "block.createframed.cyan_stained_framed_glass": "Бирюзовое витражное обрамлённое стекло",
```
This affects lines 24, 42, 60, 78, 96, 114, 132, 150, 168, 186 & 205.